docs/guides/gutenberg-blocks.md

```diff
@@ -10,10 +10,9 @@ _Note: Since the Gutenberg SDK is still being actively developed, the developmen
     yarn docker:wp plugin install gutenberg --activate
     ```
 
-1.  In Jetpack, enable loading block assets by adding following filters to your local mu-plugins folder:
+1.  In Jetpack, enable loading local block assets by adding following filter to your local mu-plugins folder:
```

Comment:
Can't we drop this line (and the other `add_filter()` statement) too, since we're now defaulting to `true` [here](https://github.com/Automattic/jetpack/pull/10292/files#diff-9ba9fa5bb52bd6cf36c6c0bdd5ed4830R7323)?

Reply:
I misinterpreted this and indeed, CDN being disabled by default makes this bit of the doc unnecessary.

I've added more cleanup to the doc as part of this PR.